app/src/main/java/com/ndadheech/android/howdy/HowdyLab.java

```diff
@@ -0,0 +1,89 @@
+package com.ndadheech.android.howdy;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.CursorWrapper;
+import android.database.sqlite.SQLiteDatabase;
+import android.util.Log;
+
+import com.ndadheech.android.howdy.database.HowdyBaseHelper;
+import com.ndadheech.android.howdy.database.HowdyDbSchema.HowdyTable;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeFieldType;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+
+public class HowdyLab {
```

Comment:
There is already a HowdyActivity. Since this class hosts the Howdy DB, maybe I can call it HowdyDao?